### ISSUE TITLE
UDF wildcard does not work

### ISSUE THREAD

**Issue by k-ryeng** [officecenter/Autotask#22]
`$AccountLocations = Get-AtwsAccountLocation -UserDefinedField @{Name = 'Solarwinds RMM Client ID'; Value = '*'}  -Like UserDefinedField`
returns 
`Get-AtwsAccountLocation : Cannot validate argument on parameter 'Like'. The argument "UserDefinedField" does not belong to the set "LocationNameUserDefinedField" specified by
 the ValidateSet attribute. Supply an argument that is in the set and then try the command again.
At line:1 char:108
+ ... me = 'Solarwinds RMM Client ID'; Value = '*'}  -Like UserDefinedField
+                                                          ~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-AtwsAccountLocation], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Get-AtwsAccountLocation`

**Comment by klemmestad** [comment]
Automatic typing has its pitfalls:
```powershell
$Labels = $Fields | Where-Object { $_.Type -eq 'string' } | Select-Object -ExpandProperty Name
If ($Entity.HasUserDefinedFields) {$Labels += 'UserDefinedField'}
```
For an entity with only a single field of type string $Labels becomes a string, not an array...

**Comment by klemmestad** [comment]
Should be fixed in module version 1.6.2.8
